Expecto/Expecto.Impl.fs

```diff
@@ -1026,17 +1026,32 @@ module Impl =
       SourceLocation.empty
 
   /// Scan filtered tests marked with TestsAttribute from an assembly
-  let testFromAssemblyWithFilter typeFilter (a: Assembly) =
+  let testListFromAssemblyWithFilter typeFilter (a : Assembly) =
     a.GetExportedTypes()
     |> Seq.filter typeFilter
     |> Seq.choose testFromType
     |> Seq.toList
+
+  /// Scan filtered tests marked with TestsAttribute from an assembly
+  let testFromAssemblyWithFilter typeFilter (assembly : Assembly) =
+    testListFromAssemblyWithFilter typeFilter assembly
+    |> listToTestListOption
```

Comment:
This looks like it just changed the return type of the function `testFromAssemblyWithFilter` from a list to whatever `listToTestListOption` returns?

Reply:
True, we could probably compose those functions together instead. 👍

Reply:
My point here is that you've changed a public ABI, by currying instead of providing explicit parameters. Could you explicitly declare and pass the parameters please to avoid this change? (Eta expansion) Thank you